spec/embedded_sass.proto

```diff
@@ -98,6 +98,13 @@ message InboundMessage {
         // generating this ID and ensuring that it's unique across all importers
         // registered for this compilation.
         uint32 file_importer_id = 3;
+
+        // The [Node.js package importer], which is a built-in Package Importer,
+        // with an associated `entry_point_url`, that resolves `pkg:` URLs using
```

Comment:
Nit: remove the commas around "with an associated `entry_point_url`".

Reply:
Updated in https://github.com/sass/sass/commit/ac235148468a3778500a37d757bef8236616b0de